main.js

```diff
@@ -0,0 +1,51 @@
+function calculator(){
+  //When number is clicked, put into top section
+  var numberButton = document.querySelectorAll('.buttonNum')
```

Comment:
Missing semicolons at the end of all relevant lines. Not a big deal, but try to get in the habit of doing it.